opensrp-giz-malawi/src/main/java/org/smartregister/giz/util/GizUtils.java

```diff
@@ -500,4 +506,141 @@ public static void createChildGrowthEventFromRepeatingGroup(@NonNull JSONObject
         interactor.processWeight(client.getIdentifiers(), tempForm.toString(), params, clientJson);
     }
 
+    public static String getDuration(DateTime dateTime) {
+        if (dateTime != null) {
+            Calendar dateCalendar = Calendar.getInstance();
+            dateCalendar.setTime(dateTime.toDate());
+            dateCalendar.set(Calendar.HOUR_OF_DAY, 0);
+            dateCalendar.set(Calendar.MINUTE, 0);
+            dateCalendar.set(Calendar.SECOND, 0);
+            dateCalendar.set(Calendar.MILLISECOND, 0);
+
+            Calendar today = Calendar.getInstance();
+            today.set(Calendar.HOUR_OF_DAY, 0);
+            today.set(Calendar.MINUTE, 0);
+            today.set(Calendar.SECOND, 0);
+            today.set(Calendar.MILLISECOND, 0);
+
+            long timeDiff = Math.abs(dateCalendar.getTimeInMillis() - today.getTimeInMillis());
+
+            DateTime todayDateTime = new DateTime();
+            return getDuration(timeDiff, dateTime, todayDateTime, getLocale());
+        }
+        return null;
+    }
+
+    private static Locale getLocale() {
+        Locale locale = CoreLibrary.getInstance().context().applicationContext().getResources().getConfiguration().locale;
+        locale = locale != null && locale.toString().startsWith("ar") ? Locale.ENGLISH : locale;
```

Comment:
Why check for ` locale.toString().startsWith("ar") `?

Reply:
Removed